content/blog/2020-pycon-india-mentorship.md

```diff
@@ -0,0 +1,25 @@
+Title: Want to perfect your proposal? The PyCon India mentorship team awaits you. 
+Date: 2020-07-27 00:34 
+Category: 2020 
+Slug: pycon-india-mentorship 
+Summary: Want to perfect your proposal? The PyCon India mentorship team awaits you.
+
+Do you have that one idea that can make the world “a better place”? (Silicon Valley Fans, Assemble!) 
+Do you wish to submit a proposal but cannot figure out the flow of your talk or how to present it? 
+We know it is not always easy to find the right words, but don’t you worry, the PyCon India Mentorship team is here just to help out with your proposal. 
+Whether this is your first time or you are a seasoned professional who wants someone to take a look at their work, the PyCon India Mentorship team is here just to help you out with your proposal. 
+
+Now there are quite a few ways to reach out to us and you are free to use any. 
+The go-to resource for prospective speakers of PyCon India, you can have a look at one of the blogs shared on the website which shares [tips on writing a “kickass” proposal](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html). 
+In case you want to know the best practices of writing a proposal or are looking to refine your proposal, a [call](https://meet.google.com/pot-jzjq-udn) is scheduled every Saturday 1500 to 1700 IST which you can join and clear out your queries on a real-time basis. 
```

Comment:
```suggestion
In case you want to know the best practices of writing a proposal or are looking to refine your proposal, a [call](https://meet.google.com/pot-jzjq-udn) is scheduled every Saturday 1500 to 1700 IST by the mentorship team which you can join and clear out your queries on a real-time basis. 
```